### PR TITLE
[Ds-536]: add new feed

### PR DESCRIPTION
**Pull request by khowen**
# JIRA Ticket
https://jiraprod.turner.com/browse/DS-555

# What does it do and why is it needed?
With the CNNMoney rebrand there is a need to create a new feed.

# How to test
- I have a test feed available at `/business-local.xml`. 
- I also have this feed set up in our test section in Google News. 
__Note: Not posting the full URL or the info on our test section here because cnn labs is public. If you need them, please let me know!__

If you would like reproducible test steps, please let me know, and I will send them to you.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,6 +1,2 @@
 save-exact=true
-<<<<<<< HEAD
-//registry.npmjs.org/:_authToken=2f9130de-6c41-4e76-a65c-1b159eea3b64
-=======
-//registry.npmjs.org/:_authToken=2f9130de-6c41-4e76-a65c-1b159eea3b64
->>>>>>> master
+//registry.npmjs.org/:_authToken=dff6bb7f-8e56-42a4-8f69-74cb728c1ee6
```
Comment (jamsyoung) [inline .npmrc]: which token is this?  is this the read-only `cnnlabs` user's token one for the `@cnnprivate` org on npmjs.org?

File: config.js [hunk 2]
```diff
@@ -95,7 +97,10 @@ let blackList = '\\/studentnews\\/,\\/videos\\/spanish\\/,fast-facts\\/index.htm
                 region: 'us-east-1',
                 bucket: 'registry.api.cnn.io'
             },
-            logConfig: (typeof process.env.CUSTOMER === 'undefined') ? null : {logzio: {tag: `cnn-google-newsstand-${nconf.get('ENVIRONMENT').toLowerCase()}`}}
+            logConfig: (typeof process.env.CUSTOMER === 'undefined') ? null : {logzio: {tag: `cnn-google-newsstand-${nconf.get('ENVIRONMENT').toLowerCase()}`}},
```
Comment (jamsyoung) [inline config.js]: We should not use `CUSTOMER`. this is a Harbor mechanism and will be going away with Harbor on Nov/15/2018

File: .npmrc (hunk 1 above)
Comment (khowen) [reply .npmrc]: This is a read only token associated with my account.

File: config.js (hunk 2 above)
Comment (khowen) [reply config.js]: This would be part of migrating away from Harbor? We would need to update this on many projects.

File: config.js [hunk 3]
```diff
@@ -95,7 +97,10 @@ let blackList = '\\/studentnews\\/,\\/videos\\/spanish\\/,fast-facts\\/index.htm
...
+            urls: {
+                business: nconf.get('BUSINESS_URLS') ? nconf.get('BUSINESS_URLS') : '\/business|success|india|media|perspectives|investing|cars|economy|energy|homes\/'
```
Comment (jamsyoung) [inline config.js]: Are these URLs? or section names? (that happen to be in the url currently.

File: package.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/jamsyoung/cnn-google-newsstand#readme",
   "engines": {
-    "node": "6.6.0"
+    "node": "9.5.0"
```
Comment (jamsyoung) [inline package.json]: mind as well go to the most current version, `10.11.0`

File: config.js (hunk 3 above)
Comment (khowen) [reply config.js]: Good point. They are sections.

File: tasks/generate-static-feed.js [hunk 5]
```diff
@@ -263,9 +263,13 @@ if (enableElectionStory === true || enableElectionStory === 'true') {
     s3Images = getImagesFromAWS();
 }
 
+// put URLs you want to test against here
 fg.urls = [
-    //  'http://www.cnn.com/2017/06/09/design/gallery/thom-browne/index.html'
-    'http://www.cnn.com/travel/article/frank-lloyd-wright-turns-150-architecture-tour-us-midwest/index.html'//  page top gallery
+    'https://ref.next.cnn.com/2018/09/14/economy/business-economy-test-article-poc/index.html',
+    'http://ref.next.cnn.com/2018/09/14/success/business-success-test-article-poc/index.html',
+    'http://ref.next.cnn.com/2018/09/14/cars/business-cars-test-article-poc/index.html'
```
Comment (jamsyoung) [inline tasks/generate-static-feed.js]: Should not use REF urls here.  Do these work outside of the VPN?

**Comment by jamsyoung** [comment]
Are the changes in PR #50 needed?  This may be a good time to roll them in.

File: .npmrc (hunk 1 above)
Comment (jamsyoung) [reply .npmrc]: Ok, we need to the use cnnlabs one incase you win the lottery. 

I'll DM it to you.

**Pull request merged**